builds/misc/images.yaml

```diff
@@ -183,7 +183,7 @@ jobs:
      # Azure IoT Edge Metrics Collector
       - template: templates/image-linux.yaml
         parameters:
-          name: Azure IoT Edge Metrics Collector
+          name: Metrics Collector (customer-facing)
```

Comment:
I chose to remove this prefix as the relevant part was getting truncated on my screen dimensions.